src/j1cli.js

```diff
@@ -367,12 +366,17 @@ async function mutateAlertRules(j1Client, rules, update) {
   }
 }
 
+//j1Client,
+//Array.isArray(data) ? data : [data],
+//program.operation
 async function mutateQuestions(j1Client, questions, operation) {
   const created = [];
   const updated = [];
   const deleted = [];
   const skipped = [];
   const results = [];
+  var questionStrings = [];
```

Comment:
We should use `const` here instead of `var`.

Reply:
Got it